QModManager/API/SMLHelper/Initializer.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace QModManager.API.SMLHelper
+{
+    using System;
+    using System.Reflection;
+    using Harmony;
+    using Patchers;
+
+    public class Initializer
+    {
+        private static HarmonyInstance harmony;
+
+        public static void Patch()
+        {
+            Logger.Initialize();
+
+            Logger.Log($"Loading v{Assembly.GetExecutingAssembly().GetName().Version}...", LogLevel.Info);
+
+            harmony = HarmonyInstance.Create("com.ahk1221.smlhelper");
```

Comment:
Will this remain the same?

Reply:
Why don't we use `"com.qmodmanager.smlhelper"`?

Reply:
Why don't we use the same harmony instance?
Also the prefix com is usually used when you have a website, in this case `qmodmanager.com`. The harmony instance that qmodmanager uses is simply `qmodmanager`

Reply:
I forgot QMM had its own Harmony instance.  
Sure. Make them the same.